src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -396,6 +400,74 @@ public void Load()
 			{
 				//ah well, we tried, no big deal, only a couple of beta testers used this old name
 			}
+
+			// Check if we need to do a one time check (perhaps migrate to a new Settings value)
+			if (OneTimeCheckVersionNumber < _currentOneTimeCheckVersionNumber)
+			{
+				DoOneTimeCheck();
+			}
+		}
+
+		private void DoOneTimeCheck()
+		{
+			// If we ever have to do another one of these besides our minimal Andika New Basic migration
+			// we should refactor this so it calls a method based on the OneTimeCheckVersionNumber
+			do
+			{
+				if(!MigrateSettingsToAndikaNewBasicFont())
+					break; // in case of failed migration
+				OneTimeCheckVersionNumber++;
+
+			} while (OneTimeCheckVersionNumber < _currentOneTimeCheckVersionNumber);
```

Comment:
Ah! I thought I had that covered! I did! Look up a few lines higher around 406 where I do:
If (OneTimeCheckVersionNumber < kCurrentOneTimeCheckVersionNumber) before calling this method.